store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/TableUtils.java

```diff
@@ -216,7 +221,63 @@ public static Connector getConnector(final String instanceName, final String zoo
         try {
             return instance.getConnector(userName, new PasswordToken(password));
         } catch (final AccumuloException | AccumuloSecurityException e) {
-            throw new StoreException("Failed to create accumulo connection", e);
+            throw new StoreException("Failed to create accumulo connection (using password-based authentication)", e);
+        }
+    }
+
+    /**
+     * Creates a connection (using Kerberos authentication) to
+     * an accumulo instance using the provided parameters
+     *
+     * @param instanceName the instance name
+     * @param zookeepers   the zoo keepers
+     * @param principal    the Kerberos principal
+     * @param keytabPath   the path to the Kerberos keytab
+     * @return A connection to an accumulo instance
+     * @throws StoreException failure to create an accumulo connection
+     */
+    public static Connector getConnectorKerberos(final String instanceName, final String zookeepers,
```

Comment:
Where is the test for this connection? is it possible to test ?

Reply:
It is not possible to test with a Unit test. This is because `MiniAccumuloCluster` doesn't support Kerberos. It is covered by the integration tests.